camera_name: kinect
learning_enabled: false
operation_state: true
